src/lib.rs

```diff
@@ -379,7 +379,13 @@ fn convert_node(el: &Element, config: &Config, path: &String) -> Option<Value> {
         for child in el.children() {
             match convert_node(child, config, &path) {
                 Some(val) => {
-                    let name = &child.name().to_string();
+                    let name = if child.prefix().is_some() {
```

Comment:
This feels awkward. There is probably a more idiomatic way of achieving the same.